deploy/docker/localkube-dind/start.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script generates the Github Release page and uploads all the binaries/etc to that page
+# This is intended to be run on a new release tag in order to generate the github release page for that release
+
+mount --make-shared /
+
+export CNI_BRIDGE_NETWORK_OFFSET="0.0.1.0"
+/dindnet &> /dev/null &
+
+
+
+mkdir -p /etc/localkube
+base=/etc/localkube/docker
+/usr/bin/dockerd \
+  --exec-root=$base.exec \
+  --graph=$base.graph \
+  --host=unix://$base.socket \
+  --pidfile=$base.pid &> /dev/null &
+
+
+mkdir -p /etc/kubernetes/manifests
+/localkube start \
+--apiserver-insecure-address=0.0.0.0 \
+--apiserver-insecure-port=8080 \
```

Comment:
The localkube container by itself does not setup certificates on the host so I don't think that we can properly use secure serving.  We can address this in another PR.